src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiChar.tsx

```diff
@@ -21,28 +21,19 @@ import {FC} from 'react';
 
 import {CSSObject} from '@emotion/react';
 
-import {messageReactionEmoji, reactionImgSize} from './EmojiImg.styles';
+type EmojiSize = 'small' | 'medium';
 
 export interface EmojiImgProps {
-  emojiUrl: string;
-  emojiName: string;
-  emojiImgSize?: CSSObject;
+  emoji: string;
+  size?: EmojiSize;
   styles?: CSSObject;
 }
 
-const EmojiImg: FC<EmojiImgProps> = ({emojiUrl, styles, emojiName, emojiImgSize = reactionImgSize}) => {
+export const EmojiChar: FC<EmojiImgProps> = ({emoji: unicode, size = 'small', styles}) => {
+  const fontSize = size === 'small' ? '.68rem' : '.81rem';
```

Comment:
@V-Gira just to be sure, you suggest using `px` instead of `rem` here, right?

Reply:
humm `--font-size-medium` is a little big. Maybe `--font-size-xsmall` then

Reply:
@V-Gira very valid review, thanks man. 
Fixed here https://github.com/wireapp/wire-webapp/pull/16331/commits/684ff2000984f46ce79a128dbf104b60409062e2